README.md

```diff
@@ -48,14 +48,31 @@ Sent traces include the following parameters:
 - attributes:
     - `iot3.core.mqtt.topic`
     - `iot3.core.mqtt.payload_size` 
-    - `iot3.core.sdk_language`
-
-Packages
---------
-
-### JSON Schema
-
-[ETSI.org][3] proposal of implementation using the [JSON][2] language (instead of ASN.1 UPER by default).
+    - `iot3.core.sdk_language` 
+
+JSON schemas
+------------
+
+As mentioned you can find in the `schemas` directory a proposal of implementation using the [JSON][2] language
+(instead of ASN.1 UPER by default) of the following [ETSI.org][3] messages:
+- CooperativeAwarenessMessage (CAM)
+- CollectivePerceptionMessage (CPM)
+- DecentralizedEmergencyNotificationMessage (DENM)
+
+But also schemas of custom messages for V2X:
+- Information (information about service instance)
+- Status (information about a device)
```

Comment:
The status message is not about _information_, it is about providing a (kind of) health condition of the instance (which might be a device). So I'd describe it as: _status of an instance_.